script/ZenithL2.s.sol

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity 0.8.26;
+
+// system contracts
+import {RollupOrders} from "../src/orders/RollupOrders.sol";
+import {RollupPassage} from "../src/passage/RollupPassage.sol";
+// simple erc20
+import {SimpleERC20} from "simple-erc20/SimpleERC20.sol";
+// utils
+import {Script, console2} from "forge-std/Script.sol";
+
+// create2 address for Permit2
+address constant PERMIT2 = 0x000000000022D473030F116dDEE9F6B43aC78BA3;
+// address of Minter with permission to mint ERC20 tokens
+address constant MINTER = 0x9999999999999999999999999999999999999999;
```

Comment:
can you document this further? not 100% sure what's going on. is this a canary that is used later to do a find/replace on the minter address?

Reply:
> is this a canary that is used later to do a find/replace on the minter address?
correct. we can also choose the minter address whenever and i can replace it in the script (rather than find/replace in the alloc json)

Reply:
similar for `OWNER_ONE` and `OWNER_TWO`, which really should be updated in code because the storage slots that Gnosis stores owners at depend on the list of owners themselves

Reply:
kind-of a nit, but this canary value is ambiguous if preceded by a 9 nibble. can we use something like `0x9988...` (only ambiguous if preceded by `0x9988`)

Reply:
how bout this https://github.com/init4tech/zenith/pull/76/commits/1cd2987a35c629a44885e30a4cbcff74efc02fa1